Source/WebCore/inspector/InspectorOverlay.cpp

```diff
@@ -2062,10 +2062,10 @@ std::optional<InspectorOverlay::Highlight::FlexHighlightOverlay> InspectorOverla
                 if (auto index = renderChildrenInDOMOrder.find(renderChild); index != notFound)
                     orderNumbers.append("Item #"_s, index + 1);
 
-                if (auto order = renderChild->style().order(); order || hasCustomOrder) {
+                if (auto order = renderChild->style().order(); !order.isZero() || hasCustomOrder) {
                     if (!orderNumbers.isEmpty())
                         orderNumbers.append('\n');
-                    orderNumbers.append("order: "_s, order);
+                    orderNumbers.append("order: "_s, order.value);
```

Comment:
Consider removing the if statement above as a drive-by optimization?
```suggestion
                    orderNumbers.append(orderNumbers.isEmpty() ? ""_s : "\n"_s, "order: "_s, order.value);
```